app.ts

```diff
@@ -493,6 +493,9 @@ async function main() {
                         console.log("Discarded (reply to another user)");
                         continue;
                     }
+                } else if ( tweet.in_reply_to_user_id !== undefined) {
```

Comment:
Shouldn't it be === undefined? 🤔

Reply:
nope - active accounts have `in_reply_to_screen_name` populated along with this field, but inactive accounts only have this field populated. Could handle them in the same condition but I thought it was helpful to display a different log message.